.github/workflows/ci.yml

```diff
@@ -20,7 +20,19 @@ jobs:
       options: --privileged
     steps:
     - uses: actions/checkout@v2
-    - name: Run storage tests
+    - name: Run storage tests not marked as root as vdsm user
+      run: ./automation/tests-storage.sh vdsm -u=vdsm
```

Comment:
Do we really need a named parameter?

I think it will much simpler to have:

    tests-storage.sh vdsm
    tests-storage.sh root

And fail the script if you call it without arguments.

Reply:
It is how we are going to use, so the proposal makes sense.

But from the conversation we had, I understood that we wanted to differentiate 4 cases:
- Running root tests as root
- Running root tests as non-root (no possible)
- Running non-root tests as root
- Running non-root tests as vdsm (or other user?)

Even we may want to run all tests (as in `make tests-storage`). We cannot differentiate 4 cases with a single argument, hence my approach. This is more flexibility than what we require at the moment, but eases to extend the pipelines in the future.

And having root as default argument (not required) I inferred it from https://github.com/oVirt/vdsm/pull/276#discussion_r913947817

What I would do is, basically, we give the user to the script if we want to run as non-root (by default run as root). And run only the tests that match (i.e., root runs root tests, non-root runs non-root tests). This was also my first attempt at this (I didn't post it).

In this case the root user cannot run non-root tests, but again, matches how is gonna be used in the pipelines, so should be fine.